apps/api/migrations/expire-at/expire-at.migration.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  MessageRepository,
+  NotificationRepository,
+  ExecutionDetailsRepository,
+  JobRepository,
+  OrganizationRepository,
+  EnvironmentRepository,
+  JobStatusEnum,
+} from '@novu/dal';
+import { addMonths } from 'date-fns';
+
+const messageRepository = new MessageRepository();
+const notificationRepository = new NotificationRepository();
+const jobRepository = new JobRepository();
+const executionDetailsRepository = new ExecutionDetailsRepository();
+const organizationRepository = new OrganizationRepository();
+const environmentRepository = new EnvironmentRepository();
+const now = Date.now();
+const expireAtOneMonth = addMonths(now, 1);
+const expireAtSixMonths = addMonths(now, 6);
```

Comment:
It could be a good idea to add some VARIANT here,
meaning instead of adding the expire at exact same time we would add diff so Mongo won't expire them all at once in the future.